### PR TITLE
PT-530 | Prevent editing event that has been published

### PR DESCRIPTION
**Pull request by Sartonon**
## Description

- Prevent editing events that have been published

### Closes
**[PT-530](https://helsinkisolutionoffice.atlassian.net/browse/PT-530):**

### CODE REVIEW

File: src/domain/event/EditEventPage.tsx [hunk 1]
```diff
@@ -201,7 +202,7 @@ const EditEventPage: React.FC = () => {
       <LoadingSpinner isLoading={loading}>
         {!!eventData ? (
           <>
-            {isEditableEvent(eventData) ? (
+            {isEditableEvent ? (
```
Comment (jorilindell) [inline src/domain/event/EditEventPage.tsx:205]: I would still use isEditableEvent(eventData) function, but change the internal logic. Just to keep maintaining little bit easier. Now ```const isEditableEvent =
    eventData?.event?.publicationStatus === PUBLICATION_STATUS.DRAFT; ``` is duplicated

File: src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx [hunk 2]
```diff
@@ -57,7 +59,7 @@ const EventDetailsButtons: React.FC<Props> = ({
         onClick={onClickLanguage}
         selectedLanguage={selectedLanguage}
       />
-      {isEditableEvent(eventData) && (
+      {isEditable && (
```
Comment (jorilindell) [inline src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx:62]: I would use isEditableEvent helper function but change it's internal logic

Comment (Sartonon) [reply src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx:62]: Good catch. Dunno what I was doing :D

**Review by jorilindell — APPROVED** [review]
Very good!

**Pull request merged**